lib/cypress/patient_zipper.rb

```diff
@@ -22,9 +22,7 @@ def export(patient)
   end
 
   class QRDAExporter
-    C3EXPORTER = GoCDATools::Export::GoExporter.instance
-    C3_1EXPORTER = GoCDATools::Export::GoExporter.instance
-    C4EXPORTER = GoCDATools::Export::GoExporter.instance
+    CAT_EXPORTER = GoCDATools::Export::GoExporter.instance
```

Comment:
Can we call this `QRDA_EXPORTER` instead? CAT on it's own doesn't really make sense